docs/pages/database-access/auto-user-provisioning/postgres.mdx

```diff
@@ -71,16 +71,85 @@ the database, which will be created automatically if it doesn't exist.
 
 ## Step 2/3. Configure a Teleport role
 
-(!docs/pages/includes/database-access/auto-user-provisioning/common-teleport-role.mdx!)
+Database permissions are associated with a Teleport role, which can either allocate predefined database roles (configured in each database) or define specific database object permissions directly. Teleport grants these permissions for the duration of the connection.
```

Comment:
We should update the previous section on what permissions the `teleport-admin` user needs when creating it.

Reply:
I've expanded the relevant section, but please check if this is satisfactory.

Reply:
I was referring to the permissions granted to `teleport-admin` in the tabs. Currently, we are only granting `createrole`, which won't be enough for db objects.